pages/index.vue

```diff
@@ -32,27 +32,41 @@
     />
   </div>
   <div class="flex gap-3">
+    <BaseButton>Default</BaseButton>
```

Comment:
It's not the best option to have these buttons here as it's not a part of how homepage should looks like 😄 Maybe you can replace some existing buttons in the app with the one created by you in the task 😎 WDYT?
There are already few places which are waiting for the button on home and fleet pages 👌

Reply:
This is was not marked for re-review yet, I am to do cleanup :)

Reply:
So I think it would be a good approach to commit or even just stage changes locally if you want to make a checkpoint for yourself, instead of pushing it to the remote 😄 I mean, it's not a request or something, but this might be good to not confuse anyone with incoming changes, as you have no guarantee that someone will not check your changes before requesting `re-review` 😄